.travis.yml

```diff
@@ -165,7 +172,7 @@ before_install:
 
 before_script:
   # login to docker at the very beginning, so we don't run into rate-limiting
-  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  #- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
I think that this line should be changed to this, so that all will work properly:
```
 - |-
    if [ -n "$DOCKER_PASSWORD" ]
    then
      echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
    fi
```

Reply:
Ok, will take care of it